### PR TITLE
fixed vaults

### PR DESCRIPTION
**Pull request by Gajesh2007**

### CODE REVIEW

**Comment by 0xngmi** [comment]
doesn't this put all the vaults into core tvl? would rather keep the GAJ pairs on pool2

**Comment by Gajesh2007** [comment]
> doesn't this put all the vaults into core tvl? would rather keep the GAJ pairs on pool2

Aperocket has done the same thing. They have put their APE in core TVL.

File: projects/GajFinance/index.js [hunk 1]
```diff
@@ -21,34 +22,90 @@ async function staking(timestamp, ethBlock, chainBlocks) {
   return balances
 }
 
-function vaults(includePool2) {
-  return async (timestamp, ethBlock, chainBlocks) => {
+async function vaultsNonNative(timestamp, ethBlock, chainBlocks) {
     const balances = {};
-    const vaults = (await fetchURL(endpoint)).data.filter(v => {
+    const vaults = (await fetchURL(nativeEndpoint)).data.filter(v => {
       const pool2 = v.vaultName.includes("GAJ")
```
Comment (0xngmi) [inline projects/GajFinance/index.js:28]: you can remove this code

File: projects/GajFinance/index.js [hunk 2]
```diff
@@ -21,34 +22,90 @@ async function staking(timestamp, ethBlock, chainBlocks) {
...
-      const chain = vault.chain.toLowerCase()
-      await sumTokensAndLPsSharedOwners(balances, [[vault.pairAddress, true]], [vault.contractAddress], chainBlocks[chain], chain, addr => `${chain}:${addr}`)
-    }
+  ).output.map((bal) => bal.output);
+  
+  
+  const lpPositions = [];
+  for (const vault of vaults) {
+      lpPositions.push({
+        token: vault.contractAddress,
```
Comment (0xngmi) [inline projects/GajFinance/index.js:46]: shouldn't this be pairAddress?

**Pull request merged**